frontend/src/api/startup.ts

```diff
@@ -0,0 +1,64 @@
+import { useQuery } from '@tanstack/react-query';
+import { useEffect, useMemo } from 'react';
+
+import { useOutlinersStore } from '@/stores/outliners';
+import { useViewStore } from '@/stores/view';
+import { useWorkspaceStore } from '@/stores/workspace';
+import { useWorldStore } from '@/stores/worlds';
+import { StartupRequest, StartupResponse } from '@/types/startup';
+import { getWorldFeatureId } from '@/utils/world';
+
+import { b6 } from './client';
+
+const getStartup = (request: StartupRequest): Promise<StartupResponse> => {
+    return b6.post('startup', null, {
+        params: request,
+    });
+};
+
+export const useStartup = () => {
+    const root = useWorkspaceStore((state) => state?.root);
```

Comment:
We should standardise the names we use here, because we have root, context, and now workspace in the codebase. Maybe we need more than one, but we should document what they're supposed to mean at least. No action needed in this CL though.